<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="welcomeLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Welcome to the Costco+ Page</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_importMembersFromFileSelection">
       <property name="text">
        <string>Import Members</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_importPurchasesFromFileSelection">
       <property name="text">
        <string>Import Purchases</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_createMember">
       <property name="text">
        <string>Create Member</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_saveAll">
       <property name="text">
        <string>Save All</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_membershipInformation">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <attribute name="title">
           <string>Membership Information</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>0</y>
             <width>791</width>
             <height>411</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>-1</number>
              </property>
              <property name="leftMargin">
               <number>16</number>
              </property>
              <property name="rightMargin">
               <number>16</number>
              </property>
              <item>
               <widget class="QTableWidget" name="MembershipInformationTable">
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>125</number>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Member Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Membership Number</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Membership Type</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Expiration Date</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Total Amount Spent</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Rebate Amount</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_salesReport">
          <attribute name="title">
           <string>Sales Report</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>0</y>
             <width>1241</width>
             <height>461</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="leftMargin">
               <number>16</number>
              </property>
              <property name="rightMargin">
               <number>16</number>
              </property>
              <item>
               <widget class="QTableWidget" name="SalesReportTable">
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>150</number>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Purchase Date</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Membership Number</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Item Purchased</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Sales Price</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Quantity Purchased</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_inventoryList">
          <attribute name="title">
           <string>Inventory List</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>0</y>
             <width>1241</width>
             <height>461</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>-1</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>-1</number>
              </property>
              <property name="leftMargin">
               <number>16</number>
              </property>
              <property name="rightMargin">
               <number>16</number>
              </property>
              <item>
               <widget class="QTableWidget" name="InventoryListTable">
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>150</number>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Item Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Item ID Number</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Item Price</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Quantity In Stock</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Quantity Sold</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCostco">
    <property name="title">
     <string>Costco+</string>
    </property>
   </widget>
   <addaction name="menuCostco"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
